server/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -1184,6 +1184,42 @@ public void test_original_query_appears_in_jobs_log() throws Exception {
         }
     }
 
+    @Test
+    public void test_foo2() throws Exception {
+        try (Connection conn = DriverManager.getConnection(url(RW), properties)) {
+            conn.createStatement().execute("create table tbl (xs float_vector(2))");
+            PreparedStatement stmt = conn.prepareStatement("insert into tbl (xs) values (?)");
+            stmt.setObject(1, new float[] { 1.2f, 1.3f });
+            stmt.executeUpdate();
+
+            conn.createStatement().execute("refresh table tbl");
+            var resultSet = conn.createStatement().executeQuery("select * from tbl");
+            assertThat(resultSet.next()).isTrue();
+            assertThat(resultSet.getArray(1).getArray()).isEqualTo(new Float[] { 1.2f, 1.3f });
+        }
+    }
+
+    @Test
+    public void test_float_vector_jdbc() throws Exception {
+        try (Connection conn = DriverManager.getConnection(url(RW), properties)) {
+            conn.createStatement().execute("create table tbl (id int, xs float[])");
+            PreparedStatement stmt = conn.prepareStatement("insert into tbl (id, xs) values (?, ?)");
```

Comment:
This is testing the float array type, not the float vector?

Reply:
apologies, I mixed up when extracting separate commits. fixing.